mitodl/bootcamp-ecommerce
# Finalized nav bar

#### What are the relevant tickets?
Closes #591 

#### What's this PR do?
- Fixes the navbar UI to more closely match the designs + design feedback
- Renders the same navbar in the React- and Django-based apps
- Fixes the positioning of notifications

#### How should this be manually tested?
- Look at the navbar in multiple screen widths, logged in and logged out. Check it against the designs
- Add a site-wide notification in the CMS. Make sure it shows up directly below the navbar

#### Any background context you want to provide?
- The "Sign In"/"Create Account"/"Sign Out" buttons are styled differently from the designs. This was based on some verbal feedback
- The "How to Apply" link will not work unless you add a resource page in the CMS entitled "Apply"
- This PR also fixes a mobile-width issue with the "View Statements" link
- ~~We might want to use an expandable menu in the mobile view like we do in xpro. That can be done in this PR or a future one~~ ~~This expandable menu was actually in an Invision design that I didn't see. Implementing that now~~ Done

#### Screenshots (if appropriate)
<img width="409" alt="ss 2020-06-18 at 14 28 53 " src="https://user-images.githubusercontent.com/14932219/85059691-11213b00-b172-11ea-986c-79eed097bbd1.png">
<img width="403" alt="ss 2020-06-18 at 14 29 03 " src="https://user-images.githubusercontent.com/14932219/85059693-12526800-b172-11ea-91c7-dad0ee82f2d2.png">
<img width="1030" alt="ss 2020-06-18 at 14 31 20 " src="https://user-images.githubusercontent.com/14932219/85059695-12eafe80-b172-11ea-88fa-99cbcb332050.png">
<img width="1365" alt="ss 2020-06-18 at 14 31 32 " src="https://user-images.githubusercontent.com/14932219/85059702-141c2b80-b172-11ea-8c6e-7825207d4d99.png">
<img width="392" alt="ss 2020-06-18 at 15 03 32 " src="https://user-images.githubusercontent.com/14932219/85061533-f603fa80-b174-11ea-9427-9a1fd4890a9b.png">
<img width="952" alt="ss 2020-06-18 at 15 03 14 " src="https://user-images.githubusercontent.com/14932219/85061523-f3090a00-b174-11ea-8eb6-a47a828bc1dc.png">
<img width="1351" alt="ss 2020-06-18 at 14 33 18 " src="https://user-images.githubusercontent.com/14932219/85059708-15e5ef00-b172-11ea-84b0-f2641d23f41f.png">
